app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -4,6 +4,15 @@
   url: proposal_url(@proposal.id)
 }) %>
 
+<%
```

Comment:
isn't this embedded Ruby useless as it is embedded with `<%` instead of `<%=` ?

Reply:
It is just as it is in Original Decidim. Check the next url.
https://github.com/decidim/decidim/blob/d94c117fd1dc890ed40084010190971c7d101fed/decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb#L8

And for example in meetigns, is the same https://github.com/decidim/decidim/blob/0.17-stable/decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

Reply:
ok, but is it correct? should we open an issue?